### PR TITLE
Algo for Weighted job Scheduling

### PR DESCRIPTION
**Pull request by shivhek25**
Solves issue #11

### CODE REVIEW

**Comment by shivhek25** [comment]
@sahilbansal17  Kindly review.

File: Algorithms/Divide-and-Conquer/weighted_job_scheduling.cpp [hunk 1]
```diff
@@ -0,0 +1,80 @@
+// Name : Shivam Singhal
+// Email ID : Shivam Singhal
+
+// Algorithm on weighted job scheduling
+
+#include <bits/stdc++.h>
```
Comment (vikasgola) [inline Algorithms/Divide-and-Conquer/weighted_job_scheduling.cpp]: Follow
[Coding Standards](  https://github.com/sahilbansal17/Competitive_Coding/wiki/Coding-Standards)

Comment (vikasgola) [reply Algorithms/Divide-and-Conquer/weighted_job_scheduling.cpp]: You can't use #include<bits/stdc++.h>

File: Algorithms/Divide-and-Conquer/weighted_job_scheduling.cpp [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+	// we take high as one less than the index because we got to check
+	// for jobs that occur before current job
+	int low = 0, high = index - 1, mid = 0 ;         
+ 	while (low <= high){
+ 		mid = (low + high) / 2;
+ 		// checking for issue
+        if (a[mid].finish <= a[index].start){
+        	if (a[mid + 1].finish > a[index].start)
+        		return mid;
+            else
```
Comment (vikasgola) [inline Algorithms/Divide-and-Conquer/weighted_job_scheduling.cpp]: Correct indentation

**Comment by shivhek25** [comment]
@vikasgola Kindly review

**Comment by shivhek25** [comment]
@aditya81070 @sahilbansal17 @vikasgola Kindly review.

**Comment by shivhek25** [comment]
I learnt how to do it from geeksforgeeks and implemented it. Is that wrong ?

**Pull request closed without merging**